tests/integration/rdb.tcl

```diff
@@ -118,16 +118,34 @@ start_server_and_kill_it [list "dir" $server_path] {
 
 start_server {} {
     test {Test FLUSHALL aborts bgsave} {
+        # 1000 keys with 1ms sleep per key shuld take 1 second
         r config set rdb-key-save-delay 1000
         r debug populate 1000
         r bgsave
         assert_equal [s rdb_bgsave_in_progress] 1
         r flushall
-        after 200
-        assert_equal [s rdb_bgsave_in_progress] 0
+        # wait half a second max
+        wait_for_condition 5 100 {
+            [s rdb_bgsave_in_progress] == 0
+        } else {
+            fail "bgsave not aborted"
```

Comment:
failed once in my daily ci (test-valgrind), look list 500ms is not enough, `wait half a second max`
https://github.com/enjoy-binbin/redis/runs/4735941719?check_suite_focus=true#step:6:2578
[err]: Test FLUSHALL aborts bgsave in tests/integration/rdb.tcl
bgsave not aborted

and also there are a typo lol
> `veirfy` that bgsave failed, by checking that the change counter is still high

Reply:
for the record (happened again): https://github.com/redis/redis/runs/4948376517?check_suite_focus=true#step:6:2501

Reply:
this failure said `ERR Background save already in progress` (not about a short timeout).
not `bgsave not aborted`
hope i solved it in #10190

i'll add another commit there to make the timeout longer